controls/radpdfviewer/features/find.md

```diff
@@ -0,0 +1,147 @@
+
+#### __[C#] Example 3: Finds the previous text in the current document using specified options__
+
+{{region radpdfviewer-find_2}}
+
+    SearchResult searchResult = this.pdfViewer.FindPrevious(sampleText, textSearchOptions);
+{{endregion}}
+
+#### __[VB.NET] Example 3: Finds the previous text in the current document using specified options__
+
+{{region radpdfviewer-find_2}}
+
+    Dim searchResult As SearchResult = Me.pdfViewer.FindPrevious(sampleText, textSearchOptions)
+{{endregion}}
+
+The **SearchResult** class holds the result from a specific search command. This class exposes several properties:
+* **NotFound**: A static property which gets a default SearchResult instance when there is no search result found.
+* **Range**: Gets the text range.
+* **Result**: Gets the result as text.
+and methods:
+* **ToString**: Returns a string that represents the current object.
+
+#### __[C#] Example 4: Check if search result is different from NotFound__
+
+{{region radpdfviewer-find_3}}
+
+    if (searchResult != SearchResult.NotFound)
+    {
+        TextRange searchResultRange = searchResult.Range;
+        string searchResultAsText = searchResult.Result;
+    }
+{{endregion}}
+
+#### __[VB.NET] Example 4: Check if search result is different from NotFound__
+
+{{region radpdfviewer-find_3}}
+
+    If searchResult <> SearchResult.NotFound Then
+        Dim searchResultRange As TextRange = searchResult.Range
+        Dim searchResultAsText As String = searchResult.Result
+    End If
+{{endregion}}
+
+The **TextSearchOptions** class holds all possible search options when invoking finding. The class has various constructors that can be used depending on the necessities. Their main difference is they are receiving different amount of parameters according to the selected options. 
+constructors:
+* **TextSearchOptions(bool caseSensitive)**
+* **TextSearchOptions(bool caseSensitive, bool useRegularExpression)**
+* **TextSearchOptions(bool caseSensitive, bool useRegularExpression, bool wholeWordsOnly)**
+
+This class exposes a static property:
+* **Default**: Gets the default text search options.
+
+    #### __[C#] Example 5: The default search options__
+    {{region radpdfviewer-find_4}}
+    
+        TextSearchOptions textSearchOptions = 
+            new TextSearchOptions(caseSensitive: false, useRegularExpression: false, wholeWordsOnly: false);
+    {{endregion}}
+    
+    #### __[VB.NET] Example 5: The default search options__
+    
+    {{region radpdfviewer-find_4}}
+    
+        Dim textSearchOptions As TextSearchOptions = 
+            New TextSearchOptions(caseSensitive:=False, useRegularExpression:=False, wholeWordsOnly:=False)
+    {{endregion}}
+
+several non-static properties: 
+* **UseRegularExpression**: Gets or sets the use regular expression option.
+* **CaseSensitive**: Gets or sets the case sensitive option.
+* **WholeWordsOnly**: Gets or sets the whole words only.
```

Comment:
These descriptions are not very meaningful. Try to provide more helpful information.